terminal/static/terminal.js

```diff
@@ -23,7 +24,7 @@ function initTerminal() {
 
 window.onunload = function () {
   navigator.sendBeacon(
-    "/shutdown",
+    `/${terminalPortString}/shutdown`,
```

Comment:
same as above, the endpoint should be `/shutdown`, not `/{uuid4}/shutdown`, I don't understand how this works ?

Reply:
We have to use url rewrite so we can multiplex terminal sessions on via port 443 to the individual terminal sessions.
This means each endpoint has to have a prefix of "terminalXXXX" where XXXX is the port number.